pyEpiabm/pyEpiabm/property/place_foi.py

```diff
@@ -35,9 +36,11 @@ def place_inf(place, infector, time: float):
             Infectiousness parameter of place
 
         """
+        transmission = PlaceInfection.params["place_transmission"]
+        place_ind = place.place_type.value - 1
+        num_groups = PlaceInfection.params["mean_group_size"][place_ind]
```

Comment:
```suggestion
        place_idx = place.place_type.value - 1
        num_groups = PlaceInfection.params["mean_group_size"][place_idx]
```
`idx` is typically the standard abbreviation for index so this is slightly more readable